### ISSUE TITLE
Console error unknown command setpixelparam on map battlefield

### ISSUE THREAD

**Issue by t3r6** [blue-nebula/base#270]
I noticed that this error is even present in RE 1.6.
`setpixelparam` seems to have been removed from RE 1.6 but somehow it was left in the battlefield map configuration.

Steps to reproduce:
launch the map and check console errors.

Possible solutions: either change the map configuration or add the `setpixelparam` alias.
Alias will probably be the better way since a lot of old maps made for RE 1.4 and earlier have this error.

**Comment by TheAssassin** [comment]
Good point. I think we should rather migrate the maps. That's how it was always done. We could publish a migration guide in our documentation.
